src/applications/burials/components/IntroductionPage.jsx

```diff
@@ -129,13 +129,14 @@ class IntroductionPage extends React.Component {
               </div>
               <p>
                 After we process your claim, you’ll get a notice in the mail
-                about the decision.
```

Comment:
Was this intentional?

Reply:
No, and thx for catching that!

Just pushed a fix.